components/brave_vpn/browser/connection/win/brave_vpn_helper/vpn_dns_handler.cc

```diff
@@ -149,7 +169,31 @@ void VpnDnsHandler::CloseWatchers() {
   periodic_timer_.Stop();
 }
 
+int VpnDnsHandler::GetWaitingIntervalBeforeExit() {
+  if (waiting_interval_before_exit_for_testing_.has_value()) {
+    return waiting_interval_before_exit_for_testing_.value();
+  }
+  return kWaitingIntervalBeforeExitSec;
+}
+
+void VpnDnsHandler::SetWaitingIntervalBeforeExitForTesting(int value) {
+  waiting_interval_before_exit_for_testing_ = value;
+}
+
+void VpnDnsHandler::PlanExit() {
```

Comment:
nit: Maybe we commonly use `ScheduleExit()`?

Reply:
renamed